NitroxModel/DataStructures/GameLogic/AbsoluteEntityCell.cs

```diff
@@ -29,11 +30,6 @@ public Int3 Center
             }
         }
 
-        protected AbsoluteEntityCell()
-        {
-            // Constructor for serialization. Has to be "protected" for json serialization.
-        }
-
```

Comment:
Will removing this be a problem for the serialisation?

Reply:
🤷 It was there in the first place for ProtoBuf

Reply:
gotta add that back...